app/models/validations/financial_validations.rb

```diff
@@ -181,22 +181,39 @@ def validate_care_home_charges(record)
 
   PROVIDER_TYPE = { 1 => :local_authority, 2 => :private_registered_provider }.freeze
   NEEDSTYPE_VALUES = { 2 => :supported_housing, 1 => :general_needs }.freeze
+  CHARGE_NAMES = { scharge: "service charge", pscharge: "personal service charge", supcharg: "support charge" }.freeze
+  FREQUENCIES = {
+    2 => "every 2 weeks",
+    3 => "every 4 weeks",
+    4 => "every calendar month",
+    9 => "weekly for 46 weeks",
+    8 => "weekly for 47 weeks",
+    7 => "weekly for 48 weeks",
+    6 => "weekly for 49 weeks",
+    5 => "weekly for 50 weeks",
+    1 => "weekly for 52 weeks",
+    10 => "weekly for 53 weeks",
+  }.freeze
```

Comment:
Can these not be question.header / question.answer_label(log) ?

Reply:
grat point, have done